CMakeLists.txt

```diff
@@ -53,6 +53,8 @@ option(KERL_INPLACE_ARMv7A "Kerl minimizing memory sizes for ARMv7A" OFF)
 option(KERL_ARMv7A_ASM "Kerl neon optimiaztion for ARMv7A" OFF)
 option(KERL_ARMv8A_ASM "Kerl neon optimiaztion for ARMv8A" OFF)
 
+# ON for POXIS system, OFF for microcontrollers
+option(USE_EMBEAR_LOGGER "Use embera logger" ON)
```

Comment:
So there's no logger for MCU right?

Reply:
No, it's simple logger for MCU. it's an option for using embera or simple logger decided by the user.